app/assets/javascripts/share.js

```diff
@@ -42,29 +42,48 @@ function pageLoad() {
     }
 
     // data picker
-    if ($('.datetimepicker')) {
-        $('.datetimepicker').datetimepicker({
-            format: 'YYYY/MM/DD HH:mm',
-            stepping: 15
-        });
-        $('.datetimepicker_deadline').datetimepicker({
-            format: 'YYYY/MM/DD'
-        });
-    };
+
+    $( "#but" ).click(function() {
+      $( "#save" ).show();
+      $( "#but" ).hide();
+      $( "#wisywyg_container" ).hide();
+      var htmlString = $( "#editor" ).html();
```

Comment:
In line 58 - the same variable but using `text()` instead of `html` what is purpose?